parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -765,6 +765,8 @@ write_files:
     net.ipv4.tcp_retries2=8
     net.core.message_burst=80
     net.core.message_cost=40
+    # To avoid node instability
+    kernel.pid_max=4194304
```

Comment:
can we move this to a drop in and add at build time? similar to https://github.com/Azure/AgentBaker/blob/master/parts/linux/cloud-init/artifacts/sysctl-d-60-CIS.conf

Reply:
for sure, are these drop ins more "hard-codey" type settings that we use for safety?

Reply:
nvm, did some research and makes much more sense now :)

Reply:
yeah mostly it's nice to put as much static stuff as possible, we could certainly improve that tbh